scripts/merkleDist/merkleTree.js

```diff
@@ -0,0 +1,132 @@
+const keccak256 = require('ethereumjs-util').keccak256;
```

Comment:
I prefer to move this file into `helper` or `src/helper` then both script and test modules can import from it

Reply:
found a library that can be used instead for merkle tree generation:
https://github.com/miguelmota/merkletreejs

Reply:
updated to using the library